stack/stack.go

```diff
@@ -0,0 +1,189 @@
+package stack
+
+import (
+	"errors"
+	"github.com/Appliscale/perun/context"
+	"github.com/Appliscale/perun/utilities"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/go-ini/ini"
+	"io/ioutil"
+	"os/user"
+	"time"
+)
+
+const dateFormat = "2006-01-02 15:04:05 MST"
+
+//prepare stackinput from template
```

Comment:
Let's expand that description and use proper English language (capital letters, full sentences, punctuation) as we are generating documentation based on that.

Reply:
Something like this?
`This function bases on "DeleteStackInput" structure and destroys stack. It uses "StackName" to choose which stack will be destroy. Before that it creates session. `

Reply:
Exactly! :+1: